pkg/validations/commandvalidations/commandvalidations.go

```diff
@@ -0,0 +1,82 @@
+package commandvalidations
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+type ValidateCreateCluster struct {
+	clusterSpec    *cluster.Spec
+	deps           *dependencies.Dependencies
+	cmdValidations interfaces.Validator
+	dockerExec     validations.DockerExecutable
+}
+
+func New(clusterSpec *cluster.Spec, deps *dependencies.Dependencies, cmdValidations interfaces.Validator, dockerExec validations.DockerExecutable) *ValidateCreateCluster {
+	return &ValidateCreateCluster{
+		clusterSpec:    clusterSpec,
+		deps:           deps,
+		cmdValidations: cmdValidations,
+		dockerExec:     dockerExec,
+	}
+}
+
+func (v *ValidateCreateCluster) ValidateCreate(ctx context.Context) error {
+	runner := validations.NewRunner()
+	runner.Register(v.generateCreateValidations(ctx)...)
+	runner.Register(v.deps.GitOpsFlux.Validations(ctx, v.clusterSpec)...)
+	err := runner.Run()
+
+	return err
+}
+
+func (v *ValidateCreateCluster) generateCreateValidations(ctx context.Context) []validations.Validation {
+	return []validations.Validation{
+		func() *validations.ValidationResult {
+			return &validations.ValidationResult{
+				Name:   "validate docker executable",
+				Err:    ValidateDockerExecutable(ctx, v.dockerExec, runtime.GOOS),
+				Silent: true,
```

Comment:
Just curious: why are we setting this to true? We don't want to log when they succeed?

Reply:
I ended up marking the validations that are silent during our `create cluster` silent during the `validate` as well.  This was to address some feedback and help keep the output from being too verbose if these validations succeed.

Reply:
This makes me think that we might not want to use the current implementation of the validation runner then and just write something that fits our need better